maintenance/fixconfig/main.go

```diff
@@ -245,6 +290,42 @@ func convertJobToSecurityJob(j *config.Presubmit, dropLabels sets.String) {
 		}
 		// NOTE: this needs to be before the bare -- and then bootstrap args so we prepend it
 		container.Args = append([]string{"--ssh=/etc/ssh-security/ssh-security"}, container.Args...)
+		
+		// check for scenario specific tweaks
+		// NOTE: jobs are remapped to their original name in bootstrap to de-dupe config
+
+		// check if we need to change staging artifact location for bazel-build and e2es
+		needGCSFlag := false
+		if jobsConfig.ScenarioForJob(originalName) == "kubernetes_bazel" {
+			for _, arg := range jobsConfig.ArgsForJob(originalName) {
+				if strings.HasPrefix(arg, "--release") {
+					needGCSFlag = true
+					break
+				}
+			}
+		}
+		needStagingFlag := false 
+		if jobsConfig.ScenarioForJob(originalName) == "kubernetes_e2e" {
+			for _, arg := range jobsConfig.ArgsForJob(originalName) {
+				if strings.HasPrefix(arg, "--stage") {
+					needStagingFlag = true
+					break
+				}
+			}
+		}
+		
+		// NOTE: these needs to be at the end and after a -- if there is none (it's a scenario arg)
+		if !endsWithScenarioArgs && (needGCSFlag || needStagingFlag) {
+			container.Args = append(container.Args, "--")
+		}
+		if needGCSFlag {
+			container.Args = append(container.Args, "--gcs=gs://kubernetes-security-prow/staging/" + j.Name)
```

Comment:
`--gcs` is default to `gs://kubernetes-release-dev/bazel` in the bazel scenario, you'll also need to check that